app/controllers/dashboards_controller.rb

```diff
@@ -10,6 +10,7 @@ def show
 
     authorize! :dashboard
     @presenter = build_presenter
+    @page_content = PageContent.find_by(page: 'home')
   end
```

Comment:
The dashboard page will display the same text as the home page (but could easily be different in the future if we wanted with this design).